_pages/en_US/homebrew-launcher-(browserhax-2020).txt

```diff
@@ -0,0 +1,66 @@
+---
+title: "Homebrew Launcher (BrowserHax 2020)"
```

Comment:
Already discussed: The name just strikes me as weird. I don't think it would necessarily be bad to call this 'browserhax' or 'new browserhax' (in reference to it being a new browserhax, not to `new-browserhax-`), since it *is* a hax of the browser.

~~or zoogie could just name this better~~ :p

Reply:
we can converse about this in crc if it comes up again